modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-0-release-note.adoc

```diff
@@ -8,6 +8,42 @@ include::ROOT:whatsnew.adoc[tag=scopes-and-collections]
 
 
 [#maint-3-1-0]
```

Comment:
This is just an unrelated re-ordering of the earlier 3.1.x relnotes?

Reply:
Yup